_includes/selector.html

```diff
@@ -513,115 +489,84 @@
                 if (this.active_pip_cuda_ver === "11.2 - 11.8") {
                     cuda_suffix = "-cu11";
                 }
+
                 var libraryToPkg = (pkg) => {
                     pkg = pkg.toLowerCase();
                     if (pkg === "cucim") return pkg;
                     return pkg + cuda_suffix;
                 }
-                var pkgs = this.active_packages.map(libraryToPkg).join(" ");
-                var isArm = this.active_arch === "arm";
-                var cupy_inst = "";
+                if (this.active_packages[0] === 'Standard') {
```

Comment:
Maybe this can be a way of enforcing at least one package? The empty install command is awkward imo.
```suggestion
                if (this.active_packages.length === 0) {
                    return "Select at least one package.";
                } else if (this.active_packages[0] === 'Standard') {
```

Reply:
Put in a different way of pre-setting cuDF if they go with `choose your own adventure` in https://github.com/rapidsai/docs/pull/414/commits/c73bfc773a76f0dc55aa253c9cd3c43b5699eec0. I try to stay away from errors whenever possible and I think the vast majority of users will have a mouse-click saved by pre-populating cuDF. 

Let me know what you think - happy to iterate more, IMO this is one of the major ways people get RAPIDS, and for new users it's their first experience with us, so it's worth the effort to get right.

Reply:
I think this should be a “yes and” solution.
- Yes, enable cudf by default like you did in c73bfc7
- Also show an error if no packages are selected (e.g. by clicking cudf off) by applying this suggesion